controllers/user.js.bak

```diff
@@ -0,0 +1,160 @@
+let User = require("../models/user");
```

Comment:
Since we use git to keep track of files, we don't need to create .bak file. Please delete.

Reply:
ok got it!